README.md

```diff
@@ -20,6 +20,126 @@ What can you find in this repo?
   * [`interfaces`](https://github.com/huggingface/huggingface_hub/tree/main/widgets/src/lib/interfaces), Typescript definition files for the Hugging Face Hub.
 * [`docs`](https://github.com/huggingface/huggingface_hub/tree/main/docs), containing the official [Hugging Face Hub documentation](https://hf.co/docs).
 
+
+## Quickstart
+
+In this quick tour, we show you how to install the library and some of its core features.
+
+### Installation
+
+To install, run:
+
+```bash
+python -m pip install -U huggingface-hub
+```
+### Extracting information from the Hub
```

Comment:
I think this is a bit duplicated with https://github.com/huggingface/huggingface_hub/tree/main/src/huggingface_hub. I like this style, though, so it might be good to modify https://github.com/huggingface/huggingface_hub/tree/main/src/huggingface_hub#api-utilities-in-hf_apipy with this ideas.

Reply:
Indeed, TIL about this new dimension to the docs :) 

cc @stevhliu you might want to use some of this PR for inspiration when you revamp the documentation for this project :)

Reply:
@osanseviero I'm happy to remove all the usage sections, but would suggest keeping the installation instructions / how to run the tests on the README. Without them, the end-user / contributor has to make some educated guesses about where to start from.